app/models/solidus_viabill/gateway.rb

```diff
@@ -7,7 +7,12 @@ module SolidusViabill
   class Gateway
     include SolidusViabill
 
-    def initialize(*args); end
+    attr_reader :api_key, :secret_key
+
+    def initialize(options)
```

Comment:
Can you make the argument optional, so that we don't have to use this odd object declaration `SolidusViabill::Gateway.new({})`

Maybe add a default value to the argument like this
`def initialize(options = {})`

So that we can use `SolidusViabill::Gateway.new` where we don't require the api and secret keys